package.json

```diff
@@ -18,13 +18,13 @@
     "migrate-test-tables": "export NODE_ENV=test && node server/migrate-tables"
   },
   "dependencies": {
-    "compression": "1.7.0",
+    "compression": "^1.7.3",
```

Comment:
not sure that we want to add a `^`? especially as we upload a `package-lock.json` anyway.

Reply:
I disagree that we need to lock the values in both the `package.json` and the `package-lock.json` - but will leave that for a new PR